rescript/types/tests/test_types_formats_transformers.py

```diff
@@ -257,6 +231,7 @@ def test_rna_fasta_format_to_dna_iterator(self):
             self.assertEqual(observed, expected)
 
 
+
 class TestDNAIteratorTransformers(RescriptTypesTestPluginBase):
```

Comment:
```suggestion
class TestDNAIteratorTransformers(RescriptTypesTestPluginBase):
```

Reply:
I realized I did not commit this particular change. It was all ready to go in my local git branch. 🤦